src/routes/buttons/buttons.jsx

```diff
@@ -0,0 +1,73 @@
+import './buttons.css'
+import { ReactComponent as Loading } from '../../assets/svg/btn-loading.svg'
```

Comment:
```suggestion
import { ReactComponent as Loading } from 'assets/svg/btn-loading.svg'
```

Reply:
@ranaintizar @mrxshinji @aftabrehan   I got an error when I made this change likewise the rest of the  suggested changes

![screencapture-localhost-3000-stories-buttons-2022-12-17-18_41_40](https://user-images.githubusercontent.com/102271199/208254491-8f521109-6b91-4a80-9f31-504b400addec.png)

Reply:
@Demandtech Remove the first slash"/" before assets.